deepmd/pt/model/atomic_model/dipole_atomic_model.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+from typing import (
+    Dict,
+)
+
+import torch
+
+from .dp_atomic_model import (
+    DPAtomicModel,
+)
+
+
+class DPDipoleAtomicModel(DPAtomicModel):
```

Comment:
we may assert the type of the fitting net at the `__inti__` of the derived atomic models.